lib/src/main/java/com/auth0/android/lock/Configuration.java

```diff
@@ -322,6 +327,14 @@ private void parseLocalOptions(Options options) {
         this.privacyURL = options.getPrivacyURL() == null ? "https://auth0.com/privacy" : options.getPrivacyURL();
     }
 
+    @StyleRes
+    public int authStyleForConnection(String strategy, String connection) {
+        if (authStyles.containsKey(connection)) {
+            return authStyles.get(connection);
+        }
+        return AuthConfig.styleForStrategy(strategy);
```

Comment:
The tests run with Robolectric. A resource folder is provided with the `styles.xml` included in the original lib. The test I wrote (`AuthConfigTest`) checks that given a `Strategy` name I get the correct Style ID ref.